src/app/Shared/Isolat.model.ts

```diff
@@ -1,52 +1,3 @@
-export interface DBentry extends Record<DBtype, string | (0|1) | number> {
-    Erreger: string;
-    BfR_Isolat_Nr: string;
-    Projektname: string;
-    ESBL_AmpC_Carb: string;
-    Spa_Typ: string;
-    Entero_Spez: string;
-    Campy_Spez: string;
-    Serovar: string;
-    Serotyp: string;
-    O_Gruppe: string;
-    H_Gruppe: string;
-    stx1: string;
-    stx2: string;
-    Shigatoxin: string;
-    eae: string;
-    e_hly_gen: string;
-    AMP_Res: 0 | 1;
-    AZI_Res: 0 | 1;
-    CHL_Res: 0 | 1;
-    CIP_Res: 0 | 1;
-    CLI_Res: string;
-    COL_Res: 0 | 1;
-    ERY_Res: string;
-    FOT_Res: 0 | 1;
-    FOX_Res: string;
-    FUS_Res: string;
-    GEN_Res: 0 | 1;
-    KAN_Res: string;
-    LZD_Res: string;
-    MERO_Res: 0 | 1;
-    MUP_Res: string;
-    NAL_Res: 0 | 1;
-    PEN_Res: string;
-    RIF_Res: string;
-    SMX_Res: 0 | 1;
-    STR_Res: string;
-    SYN_Res: string;
-    TAZ_Res: 0 | 1;
-    TET_Res: 0 | 1;
-    TGC_Res: 0 | 1;
-    TIA_Res: string;
-    TMP_Res: 0 | 1;
-    VAN_Res: string;
-    Programm: string;
-    OriginalnrDesEinsenders: string;
-    Matrix: string;
-    Land: string;
-    uniqueId: number;
-};
+export type DBtype = string;
```

Comment:
Reject: remove or rename to DBKey or something

Reply:
Done.